doc/tools/coqrst/coqdomain.py

```diff
@@ -220,10 +220,7 @@ def _add_target(self, signode, name):
             # todo: then maybe the above "if" is not needed
             names_in_subdomain = self.subdomain_data()
             if name in names_in_subdomain:
-                try:
-                    print("Duplicate", self.subdomain, "name: ", name)
-                except UnicodeEncodeError: # in CI
-                    print("*** UnicodeEncodeError")
+``               print("Duplicate", self.subdomain, "name: ", name)
```

Comment:
It doesn't build, but it's because you introduced some spurious backticks here in your last force-push.